static/src/javascripts/projects/common/modules/social/share-count.js

```diff
@@ -86,12 +86,12 @@ define([
             var url = 'http://www.theguardian.com/' + config.page.pageId;
             try {
                 ajax({
-                    url: 'https://graph.facebook.com/' + url,
+                    url: 'https://graph.facebook.com/' + url, //TODO: use recent Graph API endpoint format (versioned) https://developers.facebook.com/docs/graph-api/reference/v2.7/url
                     type: 'json',
                     method: 'get',
                     crossOrigin: true
                 }).then(function (resp) {
-                    var count = resp.shares || 0;
+                    var count = resp.share.share_count || 0;
```

Comment:
Fair point. Will be closer to actual behavior.
I still don't feel comfortable with it failing silently and us relying on CP to report this kind of issues.